apps/enc.c

```diff
@@ -563,10 +563,17 @@ static void show_ciphers(const OBJ_NAME *name, void *bio_)
 {
     BIO *bio = bio_;
     static int n;
+    const EVP_CIPHER *cipher;
 
     if (!islower((unsigned char)*name->name))
         return;
 
+    cipher = EVP_get_cipherbyname(name->name);
```

Comment:
Please put a comment here explaining that this skips unsupported modes, or something.